File: Assets/Lib/Scripts/SplashScreenToMainMenu.js
#pragma strict

@script AddComponentMenu("DN/Splash Screen To Main Menu")
@script RequireComponent(GUITexture)

var timeToWait = 3.0;

function Start () {
	/* Wait for a set amount of time, then switch to the main menu */
	yield WaitForSeconds (timeToWait);
	Application.LoadLevel("MainMenu");
}

function Update () {

}

File: Assets/Lib/Scripts/SplashScreenToMainMenu.cs
using UnityEngine;
using System.Collections;

[AddComponentMenu("DN/Splash Screen To Main Menu")]
[RequireComponent (typeof(GUITexture))]
public class SplashScreenToMainMenu : MonoBehaviour
{
	[Tooltip("Time to wait before loading main menu.")]
	[Range(0f, 100f)]
	public float
		timeToWait = 3.0f;

	IEnumerator WaitCoroutine ()
	{
		yield return new WaitForSeconds (timeToWait);
		Application.LoadLevel ("MainMenu");
	}

	void Start ()
	{
		StartCoroutine (WaitCoroutine ());
	}
}
